### DISCUSSION TITLE
Changing the position of the arrow back on iOS?

### DISCUSSION THREAD

**Discussion by lsarni** [react-navigation/react-navigation#12057] [Q&A] [answered]
We use a 25px margin on our screens, but the arrow back doesn't. Is there a way to set a margin on the header or to customize the position of the arrow back?
On Android it gets correctly aligned.

<img width="655" alt="image" src="https://github.com/react-navigation/react-navigation/assets/15196342/161edf7b-ce65-4828-bdef-391a1b23392c">

**Comment by satya164** [comment, marked as answer]
That's the native iOS style:

![image](https://github.com/react-navigation/react-navigation/assets/1174278/040de281-c42d-4ac3-8888-c5e1f37d764a)

**Comment by lsarni** [comment]
Ok, but Slack for example aligns it with their content
![image](https://github.com/react-navigation/react-navigation/assets/15196342/1b63e1e4-935b-4634-943c-caf24f098ef6)

**Reply by satya164** [reply]
It's not aligned in that screenshot. There's a few pixel differences. And it's not using the native back button.

If you don't want native styling then you can provider your own with `headerLeft` and have whatever layout you want. But it won't have native behaviour or animations.
